molecule/cluster/molecule.yml

```diff
@@ -31,21 +36,26 @@ platforms:
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
 
-
 provisioner:
   name: ansible
   lint:
     name: ansible-lint
+  options:
+    diff: True
+    v: True
```

Comment:
Why verbose output? Also why it is different from settings in `default` scenario?

Reply:
Sorry, I saw two `v` in default scenario, but I was wrong. Still why verbose?

Reply:
I think it makes reading what is happening more clear. In particular when you run the tests locally.